code/__DEFINES/say.dm

```diff
@@ -31,14 +31,14 @@
 #define MODE_WHISPER_CRIT "whispercrit"
 
 #define MODE_DEPARTMENT "department"
-#define MODE_KEY_DEPARTMENT "h"
-#define MODE_TOKEN_DEPARTMENT ":h"
+#define MODE_KEY_DEPARTMENT "d"
+#define MODE_TOKEN_DEPARTMENT ":d"
```

Comment:
I'm fine with this but you're gonna ruin a lot of people's muscle memories

Reply:
the fact that someone just accidentally shadowed one is a good indicated that we need to do something I agree